contrib/nginx/template.py

```diff
@@ -0,0 +1,72 @@
+pkgname = "nginx"
+pkgver = "1.24.0"
+pkgrel = 0
+build_style = "configure"
+configure_args = [
+    "--prefix=/etc/nginx",
+    "--sbin-path=/usr/bin/nginx",
+    "--modules-path=/usr/libexec/nginx",
+    "--conf-path=/etc/nginx/nginx.conf",
+    "--http-log-path=/var/log/nginx/access.log",
+    "--error-log-path=/var/log/nginx/error.log",
+    "--pid-path=/run/nginx.pid",
```

Comment:
--lock-path too (it defaults to logs/nginx.lock apparently)

Reply:
right, where do you suggest to save that?

Reply:
also probably make these into /run/nginx/ and put /run/nginx/ as 750 into the tmpfiles.d entry as well

Reply:
>  probably make these   

lock/pid specifically into /run/nginx